datastore/elastic_search/populate.py

```diff
@@ -194,15 +194,20 @@ def create_dictionary_data_from_file(connection, index_name, doc_type, csv_file_
         os.path.basename(csv_file_path)
 
 
-def delete_entity_by_name(connection, index_name, doc_type, entity_name, logger, **kwargs):
+def delete_entity_by_name(connection, index_name, doc_type, entity_name, logger,
+                          language_script=ENGLISH_LANG,
+                          **kwargs):
     data = {
-        'query': {
-            'term': {
-                'entity_data': {
-                    'value': entity_name
-                }
-            }
-        },
+        "query": {
+            "constant_score": {
```

Comment:
Is this necessary?
Minor: formatting and indentation

Reply:
GET gogo_entity_data/_search
{
  "query": {
    "bool": {
      "must": {
        "term": {
          "entity_data": "sg_state_list_for_two_kg"
        }
      },
      "filter": {
        "term": {
          "language_script": "hi"
        }
      }
    }
  }
}

Reply:
Is this query more appropriate?

Reply:
GET gogo_entity_data/_search
{
  "query": {
    "bool": {
      "must": {
        "term": {
          "entity_data": "sg_state_list_for_two_kg"
        }
      },
      "filter": {
        "term": {
          "language_script": "hi"
        }
      }
    }
  }
}

Reply:
Is this query form more appropraite?